app/src/main/kotlin/pl/gov/mc/protego/ui/main/DashboardActivity.kt

```diff
@@ -44,25 +35,24 @@ class DashboardActivity : BaseActivity() {
         val id = item.itemId
 
         if (id == R.id.action_menu) {
-            val fragment = supportFragmentManager.findFragmentByTag(HISTORY_PANEL_TAG)
-            if (fragment != null  && fragment is HistoryFragment) {
-                supportFragmentManager.beginTransaction()
-                    .setCustomAnimations(R.animator.slide_from_left, R.animator.slide_to_right)
-                    .remove(fragment)
-                    .commit()
-            } else {
-                supportFragmentManager.beginTransaction()
-                    .setCustomAnimations(R.animator.slide_from_left, R.animator.slide_to_right)
-                    .add(R.id.container, HistoryFragment(), HISTORY_PANEL_TAG)
-                    .commit()
-            }
+            viewModel.homeButtonPressed()
```

Comment:
Wydaje mi się, że to nie jest `home`. Dla mnie to coś bliżej panelu do wysyłania danych, informacją danych zebranych przez aplikację, etc.

Reply:
miało być `menuButtonPressed` bo tak też był przycisk nazwany. zmieniam